### PR TITLE
Loadtest dev m

### PR DESCRIPTION
**Pull request by MartinNeum**
# Description

## Deployment

N/A

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: locustfile.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
 from locust.exception import LocustError, CatchResponseError, ResponseError
 from urllib.parse import urlparse
 
+# Martin Test
```
Comment (ldahlke) [inline locustfile.py]: That's a little bit to less comment

Comment (mrtn-nm) [reply locustfile.py]: Was only a test

**Review by ldahlke — CHANGES_REQUESTED** [review]
PLease revert, this makes no sense

**Review by ldahlke — APPROVED** [review]
Looks better now

**Pull request merged**